lms/djangoapps/teams/views.py

```diff
@@ -1360,11 +1360,9 @@ def delete(self, request, team_id, username):
         return Response(status=status.HTTP_204_NO_CONTENT)
 
 
-class MembershipBulkManagementView(View):
+class MembershipBulkManagementView(GenericAPIView):
```

Comment:
Why did the base class change here? I don't see it reflected anywhere else in the PR.

Reply:
I was using a mixing that wasn't needed.